packages/react-component-library/src/enhancers/withFormik.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+
+import FieldProps from '../types/FieldProps'
+import FormProps from '../types/FormProps'
+
+export interface FormikProps {
+  field: FieldProps
+  form: FormProps
+}
+
+const withFormik = (FormComponent: React.FC<any>) => ({
```

Comment:
Don't need &lt;any&gt;

Reply:
Hmm surely I do need any because the component props are arbitrary? Not sure how else I would define a contract for arbitrary props?